includes/Models/Data/Task.php

```diff
@@ -287,7 +305,10 @@ public static function get_tasks_with_name( $task_name ) {
 		$table_name = MODULE_TASKS_TASK_TABLE_NAME;
 
 		$required_tasks = $wpdb->get_results(
-			'SELECT * FROM ' . $wpdb->prefix . $table_name . ' WHERE task_name = \'' . $task_name . '\''
+			$wpdb->prepare(
+				// phpcs:ignore
```

Comment:
We should avoid using `// phpcs:ignore` since this ignores ALL rules. I think it is probably irrelevant here since you addressed the issue. However, if there is a specific rule we are trying to intentionally ignore, we should include the rule name in the comment so that other rules aren't ignored. There are several instances of this, so I'll create a separate issue to address this.

Reply:
Issue created: https://github.com/newfold-labs/wp-module-tasks/issues/13